csharp_dotnetcore/2.EchoBot-With-Counter/EchoBotWithCounter.csproj

```diff
@@ -7,7 +7,7 @@
 
 
   <ItemGroup>
-    <Content Include="EchoBotWithCounter.bot">
+    <Content Include="BotConfiguration.bot">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
This sample doesn't have any modifications to startup.cs, as all the other samples do. 

Did you miss this one, or is this an oversight?

Reply:
Good catch!!!
Oversight